src/zope/testrunner/tests/testrunner-layers-instances.txt

```diff
@@ -0,0 +1,183 @@
+Layers implemented via object instances
+=======================================
+
+Layers are generally implemented as classes using class methods, but
+regular objects can be used as well.  They need to provide __module__,
+__name__, and __bases__ attributes.
+
+>>> class TestLayer:
```

Comment:
Should we declare that we implement `IFullTestLayer`?  (I realize the runner doesn't check it)